spec/paginator/backend_helpers_spec.cr

```diff
@@ -27,6 +27,10 @@ class Paginatable
     paginate(FakeUser::BaseQuery.new)
   end
 
+  def call_array
+    paginate_array(FakeUser::BaseQuery.new.to_a)
```

Comment:
Looks like your spec fails because this actually makes the query to the DB, but this DB doesn't actually exist. Since this is just for an array, we should be able to make this a static array. If it works for static array, then it should work for a query turned into an array.

Alternatively, we would have to setup a real database for this project.

Reply:
Good catch and good suggestion! I think a static array would be a great fit here. I'd prefer not add a whole database setup for just these tests

```suggestion
    paginate_array([1,2,3])
```

Reply:
Or if you need a ton of items, `[1] * 50`

Reply:
I chose `[1]*50` - Seems to be a nice way to make it work for all tests. Thanks!